plugins/reporters/static-html/src/main/kotlin/StaticHtmlReporter.kt

```diff
@@ -389,83 +389,79 @@ class StaticHtmlReporter : Reporter {
     private fun DIV.projectTable(table: ProjectTable) {
         val excludedClass = "excluded".takeIf { table.isExcluded() }.orEmpty()
 
-        h2 {
+        div("project $excludedClass") {
```

Comment:
Commit message:
- "One dedicated column for each, open and resolved / excluded issues." -> "One dedicated column for each open, resolved, and excluded issues."
- Typo in "vizually" -> "visually".

Reply:
Now there's a typo in "resolvedm,".